libnavigation-core/src/main/java/com/mapbox/navigation/core/telemetry/TelemetryLocationAndProgressDispatcher.kt

```diff
@@ -25,7 +25,7 @@ import kotlinx.coroutines.channels.ReceiveChannel
 import kotlinx.coroutines.launch
 
 private const val DISPATCHER_TAG = "TELEMETRY_TEST"
-typealias RouteProgressReference = (RouteProgress) -> Unit
+internal typealias RouteProgressReference = (RouteProgress) -> Unit
```

Comment:
This can even be `private` right? BTW great catch!

Reply:
fixed